react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,43 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
+  renderSquare = (index, value) => <Square value={value} onClick={() => this.props.handleClick(index)} />;
 
   render() {
-    const status = 'Next player: X';
+    const { squares, status } = this.props;
+    const { boardRow } = styles;
     return (
-      <div>
+      <Fragment>
         <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
+        <div className={boardRow}>
+          {this.renderSquare(0, squares[0])}
```

Comment:
Notice that you are using the index twice, and repeating this call three times per row. I would define it once and reuse it.
To start with, you can simplify `renderSquare` to receive only the index, and use this.props.squares by itself.

Reply:
Notice that you are repeating three times this structure. I'd create a function that do it for we
```
<div className={styles.boardRow}>
    {this.renderSquare(3)}	          
    {this.renderSquare(4)}	          
    {this.renderSquare(5)}	          
  </div>
```